### ISSUE TITLE
2.62.	Outstanding:MeD: Question Cannot access radio button (WCAG 2.1.1)

### ISSUE THREAD

**Issue by falcochip** [MeD-DMC/Snap-framework-MeD#73]
m2-22, m2-23

After the user answers the first question on the page and selects the Submit button using the keyboard, they are not able to tab into the next question. Pressing the Tab key takes the user to the Submit button of the next and subsequent questions. 
We tried to press the up or down arrow keys when the keyboard focus was on the next question’s Submit button. What was observed was that page either moved up or down but we weren’t able to reach the radio buttons associated with the remaining questions using the keyboard. 
Note: We used NVDA with Chrome and noticed that the above observation did not happen when using a screen reader. This still, however, would be a problem for those that are using only the keyboard to navigate the course.
Note: We used JAWS with IE11 and noticed that we could user the arrow keys to navigate to and select the radio buttons. But the tab button still goes straight to the submit button (skipping the radio buttons).
